### PR TITLE
Create product pricing badges, apply across site

### PR DESCRIPTION
**Pull request by jwitz**
Resolves https://github.com/astronomer/docs/issues/3379

### CODE REVIEW

**Comment by jwitz** [comment]
@katiebrady-astro This hasn't occurred naturally yet, but this is what it looks like when two badge types are on the same topic:

Screenshot 2024-03-21 at 1 10 00 PM

Do you think these should appear on the same line and , if so, how would we change the CSS styling to support that?

File: astro/deployment-details.md [hunk 1]
```diff
@@ -39,6 +41,8 @@ In addition to alert emails for your Deployments, Astronomer recommends configur
 
 ## Enforce CI/CD deploys
 
+<BusinessBadge/>
+
```
Comment (jwitz) [inline astro/deployment-details.md:45]:
```suggestion
```

File: astro/manage-teams.md [hunk 2]
```diff
@@ -5,6 +5,10 @@ id: manage-teams
 description: Create, delete, and update Teams on Astro.
 ---
 
+import BusinessBadge from '@site/src/components/BusinessBadge';
+
+<BusinessBadge/>
+
```
Comment (jwitz) [inline astro/manage-teams.md:11]:
```suggestion
```

File: astro/authorize-workspaces-to-a-cluster.md [hunk 3]
```diff
@@ -5,6 +5,10 @@ id: authorize-workspaces-to-a-cluster
 description: Learn how to configure a cluster so that only specific Workspaces can use it.
 ---
 
+import EnterpriseBadge from '@site/src/components/EnterpriseBadge';
+
+<EnterpriseBadge/>
+
```
Comment (jwitz) [inline astro/authorize-workspaces-to-a-cluster.md:11]:
```suggestion
```

File: astro/audit-logs.md [hunk 4]
```diff
@@ -4,6 +4,10 @@ id: audit-logs
 sidebar_label: Export audit logs
 ---
 
+import TeamBadge from '@site/src/components/TeamBadge';
+
+<TeamBadge/>
+
```
Comment (jwitz) [inline astro/audit-logs.md:10]:
```suggestion
```

**Comment by katiebrady-astro** [comment]
> @katiebrady-astro This hasn't occurred naturally yet, but this is what it looks like when two badge types are on the same topic:
> 
> Screenshot 2024-03-21 at 1 10 00 PM
> 
> Do you think these should appear on the same line and , if so, how would we change the CSS styling to support that?

Hey @jwitz ! We can totally have them display inline. Went ahead and made a quick change to `display: inline-block` for all badges and should be good to go now! 🙌 

image

**Review by katiebrady-astro — APPROVED** [review]

File: src/components/TeamBadge/index.js [hunk 5]
```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import Link from '@docusaurus/Link';
+import styles from './styles.module.css';
+
+export default function LinkCard() {
+  return (
+    <Link className={styles.teamBadge} id="teamBadge">
+        <div className={styles.teamBadge}>
+        <a target="_blank" href="https://docs.astronomer.io/astro/astro-architecture">
```
Comment (jwitz) [inline src/components/TeamBadge/index.js:9]:
```suggestion
        <a target="_blank" href="https://www.astronomer.io/pricing/">
```

File: src/components/EnterpriseBadge/index.js [hunk 6]
```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import Link from '@docusaurus/Link';
+import styles from './styles.module.css';
+
+export default function LinkCard() {
+  return (
+    <Link className={styles.enterpriseBadge} id="enterpriseBadge">
+        <div className={styles.enterpriseBadge}>
+        <a target="_blank" href="https://docs.astronomer.io/astro/astro-architecture">
```
Comment (jwitz) [inline src/components/EnterpriseBadge/index.js:9]:
```suggestion
        <a target="_blank" href="https://www.astronomer.io/pricing/">
```

File: src/components/BusinessBadge/index.js [hunk 7]
```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import Link from '@docusaurus/Link';
+import styles from './styles.module.css';
+
+export default function LinkCard() {
+  return (
+    <Link className={styles.businessBadge} id="businessBadge">
+        <div className={styles.businessBadge}>
+        <a target="_blank" href="https://docs.astronomer.io/astro/astro-architecture">
```
Comment (jwitz) [inline src/components/BusinessBadge/index.js:9]:
```suggestion
        <a target="_blank" href="https://www.astronomer.io/pricing/">
```

**Pull request merged**